test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -927,7 +927,7 @@ class PBSServerLog(PBSLogAnalyzer):
     server_run_tag = re.compile(tm_re + ".*" + job_re + ".*;Job Run at.*")
     server_nodeup_tag = re.compile(tm_re + ".*Node;.*;node up.*")
     server_enquejob_tag = re.compile(tm_re + ".*" + job_re +
-                                     ".*enqueuing into.*state 1 .*")
+                                     ".*enqueuing into.*state R .*")
```

Comment:
@agrawalravi90 Can you please confirm that we are not changing anything in accounting log format or value like this? if changes like also visible in accounting log then it may cause problem in tools/products around accounting log...

Reply:
So, I went through the accounting chapter in our guides, the job_state is not recorded for any accounting log (check the entry for job_state):

![Screen Shot 2020-09-08 at 5 11 53 PM](https://user-images.githubusercontent.com/4574875/92528235-6d867a80-f1f6-11ea-95b3-71e4ea552080.png)

So, I feel that it should be fine. I also passed the accounting logs to some of our tools which consume them and they ran ok